examples/addresses/create_address.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"encoding/json"
+	"github.com/EasyPost/easypost-go"
+)
+
+func main() {
+	apiKey := os.Getenv("EASYPOST_API_KEY")
+	if apiKey == "" {
+		fmt.Fprintln(os.Stderr, "missing API key")
+		os.Exit(1)
+		return
+	}
+	client := easypost.New(apiKey)
+
+	// Create and verify an address
+	toAddress, err := client.CreateAddress(
+		&easypost.Address{
+			Name:    "Bugs Bunny",
+			Street1: "4000 Warner Blvd",
+			City:    "Burbank",
+			State:   "CA",
+			Zip:     "91522",
+		},
+		&easypost.CreateAddressOptions{Verify: []string{"delivery"}},
+	)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "error creating to address:", err)
+		os.Exit(1)
+		return
+	}
+
+	prettyJSON, err := json.MarshalIndent(toAddress, "", "    ")
+    if err != nil {
+		fmt.Fprintln(os.Stderr, "error creating JSON:", err)
+    }
```

Comment:
This formatting appears to be off in all the files.

```suggestion
	if err != nil {
		fmt.Fprintln(os.Stderr, "error creating JSON:", err)
	}
```

Reply:
Is this only off on this particular if statement? Shows as indented with 4 spaces in VS Code but after re-indenting it I found it was lying.